src/functionBased/components/InputTodo.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable react/button-has-type */
```

Comment:
Hey Krishna
Great job implementing most of the project requirements and following the steps🙂.
 Descriptive pull request ✔️
 Linters ✔️
Optional changes!
However your week has ended and you might not be able to make a resubmission so lm adding most of the changes under optional changes.

For the next project.l suggest you do not disable eslint several times like in this case, l think it was best to add type attributes to your buttons this is because it is needed to check for any bugs.